lib/reek/smells/smell_detector.rb

```diff
@@ -83,14 +81,11 @@ def register(hooks)
       end
 
       # SMELL: Getter (only used in 1 test)
+      # MvZ: Or use it more :-).
```

Comment:
Hm, seems like there's lots of weirdness going on with the config stuff. I'd like to unify this so it always goes through a SmellConfiguration object rather than having the current mix of SmellConfiguration and Hash.